testsuites/kubernetes_versions.robot

```diff
@@ -27,42 +27,33 @@ Documentation     Verify Helm functionality on multiple Kubernetes versions.
 ...
 Library           String
 Library           OperatingSystem
-Library           ../lib/Kind.py
 Library           ../lib/Kubectl.py
 Library           ../lib/Helm.py
 Library           ../lib/Sh.py
-Suite Setup       Suite Setup
+Library           ../lib/ClusterProvider.py
 Suite Teardown    Suite Teardown
 
 *** Test Cases ***
-#Helm works with Kubernetes 1.16.1
-#    Test Helm on Kubernetes version   1.16.1
 
-Helm works with Kubernetes 1.15.3
-    Test Helm on Kubernetes version   1.15.3
+Helm Works on Kubernetes
+    Test Helm on Kubernetes version
 
-Helm works with Kubernetes 1.14.6
-    Test Helm on Kubernetes version   1.14.6
 
 *** Keyword ***
+
 Test Helm on Kubernetes version
     Require cluster  True
-
     ${helm_version} =  Get Environment Variable  ROBOT_HELM_V3  "v2"
     Pass Execution If  ${helm_version} == 'v2'  Helm v2 not supported. Skipping test.
-
-    [Arguments]    ${kube_version}
-    Create test cluster with kube version    ${kube_version}
-
+    Create test cluster with kube version   %{CLUSTER_VERSION}
```

Comment:
why do not we make keyword that accept arg and use the env var above?

Reply:
Done